### PR TITLE
Update complaint table with sortable incident date

### PR DESCRIPTION
**Pull request by el-mapache**
[Link to ZenHub issue.](https://github.com/18F/crt-portal/issues/236)

## What does this change?
Adds incident date as a sortable column!

* Removes protected class column
* Adds 'ascending' and 'descending' to screen reader labeling

## Screenshots (for front-end PR):
Screen Shot 2020-02-03 at 11 22 32 AM

### Author

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] Check for [accessibility](/docs/a11y_plan.md).
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

### Reviewer

+ [ ] If this is a story, run locally and check to make sure all Acceptance Criteria are met. (If any criteria are unclear, ask about them.)
+ [ ] [Tests pass](https://github.com/USDOJ/crt-portal/#tests).

## Notes for reviewer:

I have an open question on the behavior of this feature! Currently when you sort, if `last_incident_day` is missing, those dates will appear after any records that have complete dates filled in i.e. those with M/D/YYYY.

I'm not 100% is this behavior is what the intake specialist would want, but would like to hear your thoughts!

See [PR instructions doc](https://github.com/usdoj/crt-portal/blob/master/docs/pull_requests.md) for full pull request review instructions.

### CODE REVIEW

File: crt_portal/cts_forms/templates/forms/snippets/sortable_table_heading.html [hunk 1]
```diff
@@ -6,7 +6,7 @@
       id="{{id}}"
       href="{{sort_url}}{{filter_state}}"
       class="sort-link display-block {% if is_descending %}sort-up{% else %}sort-down{% endif %}"
-      aria-label="Sort complaints by property {{ heading }}"
+      aria-label="Sort complaints by property {{ heading }} {% if is_descending %}descending{% else %}ascending{% endif %}"
```
Comment (LindsayYoung) [inline crt_portal/cts_forms/templates/forms/snippets/sortable_table_heading.html]: Nice!!

**Comment by LindsayYoung** [comment]
This changes the default sort to incident date, but I don't know why - the default sort on line 42 views.py hasn't changed:
https://github.com/usdoj-crt/crt-portal/blob/ab-236/crt_portal/cts_forms/views.py#L42

Also, this issue included the date on the detail view page, but that's not in this PR

**Comment by el-mapache** [comment]
That's super weird, I'll look at it, I definitely added nothing to the other page, I was lazily thinking that we were just updating this page in this ticket 😢 😬

**Comment by el-mapache** [comment]
@LindsayYoung Can you describe under what conditions you see the date fields as the new default sort? I'm not seeing that behavior. The steps I'm following are:

* Loading the complaints table
* Results are displayed according to descending sort date
* Clicking a record
* Clicking the `close record` button, I see that in the url `create-date` is still the default sort

Any extra context would be great! Thanks!

**Comment by LindsayYoung** [comment]
No problem! I will double check can take some screen shots.

**Comment by LindsayYoung** [comment]
I can't reproduce the sorting issue. I don't know what happened before. Sorry to waste your time.

**Pull request merged**